MapboxNavigation.podspec

```diff
@@ -2,12 +2,12 @@ Pod::Spec.new do |s|
 
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
-  s.name = "MapboxNavigationUI.swift"
-  s.version = "0.1.0"
+  s.name = "MapboxNavigation"
+  s.version = "0.2.0"
   s.summary = "Mapbox Navigation UI library"
 
   s.description  = <<-DESC
-  MapboxNavigationUI.swift makes it easy to get step by step UI for guiding a user along a route.
+  MapboxNavigation makes it easy to get step by step UI for guiding a user along a route.
```

Comment:
Let’s refer to this library as “Mapbox Navigation” or “The Mapbox Navigation SDK” in prose.